crates/stages/src/pipeline/mod.rs

```diff
@@ -682,15 +680,15 @@ mod tests {
         let db = test_utils::create_test_db::<mdbx::WriteMap>(EnvKind::RW);
         let mut pipeline: Pipeline<_, NoopSyncStateUpdate> = Pipeline::builder()
             .add_stage(TestStage::new(StageId("Fatal")).add_exec(Err(
-                StageError::DatabaseIntegrity(DatabaseIntegrityError::BlockBody { number: 5 }),
+                StageError::DatabaseIntegrity(ProviderError::BlockBody { number: 5 }),
```

Comment:
Probably `StageError::DatabaseIntegrity` should be renamed as well now? Todo in followup